images/builder/docker/sti-image-builder/Dockerfile

```diff
@@ -0,0 +1,14 @@
+FROM centos:centos7
+
+RUN yum install -y --enablerepo=centosplus epel-release gettext tar automake \
+      make git docker
+
+# TODO: STI should be installed from official release page, instead of bundling
+#       it in the GIT repository.
+ADD bin/sti.gz /usr/bin/sti.gz
+RUN gunzip /usr/bin/sti.gz
```

Comment:
Given that we vendor STI into origin, i think it would be better if we built and embedded the sti that's in there, which is effectively what we do for the sti-builder image.

that way we are more self contained.